components/multi-platform-controller/staging/host-config.yaml

```diff
@@ -7,35 +7,211 @@ metadata:
   namespace: multi-platform-controller
 data:
 
-  dynamic-platforms: linux/arm64,linux/amd64,linux-root/arm64,linux-root/amd64
+  # dynamic-platforms: linux/arm64,linux/amd64,linux-root/arm64,linux-root/amd64
```

Comment:
let's remove the commented line, we can get it back from git history if needed

Reply:
commented line removed :)